python/graphscope/__init__.py

```diff
@@ -29,3 +29,22 @@
 from graphscope.framework.graph_utils import g
 from graphscope.framework.graph_utils import load_from
 from graphscope.version import __version__
+
+# module level doc-string
+__doc__ = """
+GraphScope - A unified distributed graph computing platform
+=====================================================================
+
+GraphScope provides a one-stop environment for performing diverse graph operations on a cluster of computers through a user-friendly Python interface. GraphScope makes multi-staged processing of large-scale graph data on compute clusters simple by combining several important pieces of Alibaba technology: including GRAPE, MaxGraph, and Graph-Learn (GL) for analytics, interactive, and graph neural networks (GNN) computation, respectively, and the vineyard store that offers efficient in-memory data transfers.
```

Comment:
break line into at maximum 80/120 chars every line for better help view in console/notebook.